tempesta_fw/t/unit/test_tfw_str.c

```diff
@@ -61,36 +62,36 @@ TEST(tfw_strcpy, both_plain)
 	char buf1[4] = { 0 }, buf2[4] = "abc";
 	TfwStr s1 = {
 		.len = 4,
-		.ptr = buf1
+		.data = buf1
 	};
 	TfwStr s2 = {
 		.len = 4,
-		.ptr = buf2
+		.data = buf2
 	};
-
+	TFW_DBG("test:plain:cpy\n");
 	EXPECT_ZERO(tfw_strcpy(&s1, &s2));
-	EXPECT_STR_EQ(s1.ptr, "abc");
+	EXPECT_STR_EQ(s1.data, "abc");
 }
 
 TEST(tfw_strcpy, src_compound)
 {
 	char buf1[32] = { 0 };
 	TfwStr s1 = {
 		.len = 32,
-		.ptr = buf1
+		.data = buf1
```

Comment:
There are macros `DEFINE_TFW_STR()` and `TFW_STR_FROM()` that can be used for this purpose.